src/dsl/safe-types.ts

```
@@ -0,0 +1,369 @@
+    case 'compare_property':
+    case 'type_check':
+    case 'always':
+      return ComplexityClass.CONSTANT;
+
+    case 'not':
+      return getPredicateComplexity(predicate.predicate);
+
+    case 'and':
+    case 'or':
+      // Max complexity of all sub-predicates
+      const complexities = predicate.predicates.map(getPredicateComplexity);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)